beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -1131,6 +1135,34 @@ impl<'a, T: BeaconChainTypes> FullyVerifiedBlock<'a, T> {
         // `randao` may change.
         let payload_verification_status = notify_new_payload(chain, &state, block.message())?;
 
+        if payload_verification_status == PayloadVerificationStatus::NotVerified {
```

Comment:
I've made a PR to your branch over in https://github.com/ethDreamer/lighthouse/pull/5 to include a new import condition I'm adding in https://github.com/ethereum/consensus-specs/pull/2841.

Reply:
merged!